app/javascript/dashboard/routes/dashboard/settings/profile/ChangePassword.vue

```diff
@@ -125,10 +125,9 @@ export default {
         });
         this.errorMessage = this.$t('PROFILE_SETTINGS.PASSWORD_UPDATE_SUCCESS');
       } catch (error) {
-        this.errorMessage = this.$t('RESET_PASSWORD.API.ERROR_MESSAGE');
-        if (error?.response?.data?.message) {
-          this.errorMessage = error.response.data.message;
-        }
+        this.errorMessage =
```

Comment:
Can't we use `parseAPIErrorResponse` (store/utils/api)?

Reply:
Yes, we can use `parseAPIErrorResponse `. Updated the PR https://github.com/chatwoot/chatwoot/pull/9114/commits/1f2ce7c92942f710e469a0130002481a2d075d45